packages/sirv-cli/index.js

```diff
@@ -58,6 +58,7 @@ module.exports = function (dir, opts) {
 		opts.cert = readFileSync(opts.cert);
 		if (opts.cacert) opts.cacert = readFileSync(opts.cacert);
 		if (opts.pass) opts.passphrase = opts.pass;
+		if (opts.allowHTTP1 === undefined) opts.allowHTTP1 = true;
```

Comment:
Can you remove this? Unless I'm mistaken, there's no need to turn this off ever.
```suggestion
```